kinto/core/resource/__init__.py

```diff
@@ -12,7 +12,7 @@
                                     HTTPNotFound, HTTPServiceUnavailable)
 
 from kinto.core import Service
-from kinto.core.errors import http_error, raise_invalid, send_alert, ERRORS
+from kinto.core.errors import http_error, raise_invalid, send_alert, ERRORS, request_GET
```

Comment:
I indeed put it in utils first but then I had a recursive import error when using http_error and ERRORS from utils.